tests/test_views.py

```diff
@@ -35,6 +35,11 @@ def setUp(self):
     def testServer(self):
         self.assertEqual(404, self.app.get('/').status_code)
 
+    def testCors(self):
+        response = self.app.get('/')
```

Comment:
Should we test POST and OPTIONS here as well? Should we look at all the other endpoints too (not just '/')? I don't really understand CORS I'm afraid, so sorry if these are stupid questions!